web/src/util/browser.js

```diff
@@ -10,7 +10,7 @@ if (browser && browser.version) {
 // well as all visual and style features to be correct.
 const browserIsGreen =
   // Current major version as of April 2020.
```

Comment:
Super minor: can you update this comment to note that we're intentionally supporting a slightly older Chrome and why? Just so future Paul or future Richard knows we knew Chrome 78 wasn't the most recent in April 2020. 😛